pkgs/by-name/do/dooit/package.nix

```diff
@@ -5,53 +5,62 @@
   python311,
   testers,
   nix-update-script,
+  extraPackages ? [ ],
```

Comment:
What is the purpose of introducing this?

Reply:
So the config file for dooit is in python and dooit now supports plugins which wont be possible unless the libraries that are being used are in the same space? See [Installation](https://dooit-org.github.io/dooit/getting_started/installation.html#flakes)

Also I mailed you regarding this. I think it got lost :)

Reply:
Oh, shoot... I think I remember seeing a notification for that a long time ago... sorry. I'll take a look closer at that new functionality.

Reply:
No problemo! I eventually got that working. It was so frustrating but it worked out in the end :)